bigbluebutton-html5/imports/ui/components/tabs/Tabs.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+
+import Tab from './components/Tab';
+import TabContentPpt from './components/TabContentPpt';
+import TabContentPdf from './components/TabContentPdf';
+import TabContentVideo from './components/TabContentVideo';
+import TabContentWeb from './components/TabContentWeb';
+
+
+const Tabs = () => {
+  const [key, setKey] = useState(1);
+  return (
+    <>
+      <aside className="primary-nav w-1/12">
+        <div className="h-24 bg-green-900 rounded-lg text-white font-bold text-3xl justify-center items-center flex m-3">SeeIT</div>
+
+        <ul className="flex flex-col justify-center items-center">
+          <Tab
+            index={1}
+            icon="ppt"
+            activeKey={key}
+            tabArea="#Link1"
+            onClickEvent={(e) => {
+              e.preventDefault();
+              setKey(1);
```

Comment:
I would suggest creating a common function above render

```
const handleTabClick = (index) => {
  currentTab(index)
}
```

and use it like

`onClick={handleTabClick}`

i think no need to add `e.preventDefault()`

Reply:
done